week-3.md

```diff
@@ -0,0 +1,63 @@
+# Week Three Info
+
+> "Don't roll your own crypto, bro." -- Joseph Cox, VICE.com [2015-12-11](https://www.vice.com/en_us/article/wnx8nq/why-you-dont-roll-your-own-crypto)
+
+With luck, everyone's into the swing of things now, because we're moving on to the first look at the final BIP, both the alpha and omega, if you will: [bip-schnorr](https://github.com/sipa/bips/blob/bip-schnorr/bip-schnorr.mediawiki). This marks the halfway point -- by the end of this week we'll have gone through all the BIPs (if you're following the [curriculum](Curriculum.md) anyway!) and for the following three weeks we'll be focussing on some of the details (that you might have skipped over if you followed the suggestions here) and their implications.
+
+The quote this week emphasises the widely known wisdom that you shouldn't roll your own crypto -- even though that's essentially what we're doing here with both a new standard for schnorr signatures, and introducing the taproot construction for real world use. Both of these are security constructions that rely on mathematical assumptions working out as expected; so before adopting it, it's wise to make sure that these new constructions really can survive expert cryptanalysis. (We'll see in the "schnorr applications" week how the first version of the musig construction didn't survive expert review, eg)
```

Comment:
nit: missing period at end of sentence and could probably remove ", eg" or replace it with s/We'll/For instance, we'll/